app/locale/fr.coffee

```diff
@@ -719,7 +719,7 @@ module.exports = nativeDescription: "français", englishDescription: "French", t
     purchased: "Acheté"
 
 #  subscribe_for_gems:
```

Comment:
Please uncomment this line. This will make your `prompt_title` translation work.